UPGRADE.md

```diff
@@ -1,5 +1,65 @@
 # Upgrade
 
+## unreleased
+
+### Sitemap Provider changed
+
+As a sitemap is always domain specific and a domain can have multiple webspaces and portal
+the `SitemapProviderInterface` changed. To make it possible to provide also none Sulu
+routes the `SitemapUrl` constructor changed and now needs as third paramter a defaultLocale.
```

Comment:
Use backticks for `defaultLocale` and you've got a typo in "parameter". I would also change the sentence to "constructor introduced a `defaultLocale` as third parameter"